### DISCUSSION TITLE
UR ROS2 + Unity

### DISCUSSION THREAD

**Discussion by RafaelBrunet** [UniversalRobots/Universal_Robots_ROS2_Driver#357] [Q&A] [answered]
Hi,

I'm tryning to import a UR10e in unity as a digital twin. I have try to follow guides such as:
https://github.com/Unity-Technologies/URDF-Importer
But there seems that there are no URDF files like in ROS1.

Do you know how can I import a robot using ROS2?

Thanks in advance!

**Comment by fmauch** [comment, marked as answer]
for the urdf files, please have a look at [`ur_description`](https://github.com/UniversalRobots/Universal_Robots_ROS2_Description). I hope, this helps!

**Reply by RafaelBrunet** [reply]
Thanks a lot! That's helpful!

Do you have guide of how to set the xacro file?

**Reply by fmauch** [reply]
You can take a look into the ur_bringup launchfiles. There you will find according xacro commands.

**Reply by RafaelBrunet** [reply]
Thanks a lot for your quick are great responses.

I'll have a look!
